blaise82/l.m.s-server
# #170879950 Setup the database schema

### What does this PR do?
Have the models and the migration files configured

### Description of task to be completed?
- Install sequelize js and all the related dependencies.
- Configure the config.js file that contains the database information.
- Create and configure the models concerning their relationships.
- Configure the migration files.

### How should this be manually tested?
- Clone this repo to your local repository using `git clone <repo link>`
- Install all project dependencies using `npm i`
- Create a .env file and add your database information(username, password, host)
- run `sequelize db:migrate` and check if the tables are created in your database.

### Any background context you want to provide?

### What are the relevant pivotal tracker stories?

[#170879950](https://www.pivotaltracker.com/story/show/170879950)
